apps/test/unit/lib/levelbuilder/lesson-editor/LessonEditorTest.js

```diff
@@ -22,12 +22,16 @@ describe('LessonEditor', () => {
 
   it('renders default props', () => {
     const wrapper = shallow(<LessonEditor {...defaultProps} />);
-    expect(wrapper.contains('Lesson Name'));
-    expect(wrapper.contains('Lesson Overview'));
-    expect(wrapper.contains('Overview of the lesson for students'));
-    expect(
-      wrapper.contains('The purpose of the lesson is for people to learn')
-    );
+    expect(wrapper.contains('Lesson Name'), 'Lesson Name').to.be.true;
+    // expect(wrapper.contains('Lesson Overview'), 'Lesson Overview').to.be.true;
+    // expect(
+    //   wrapper.contains('Overview of the lesson for students'),
+    //   'student overview'
+    // ).to.be.true;
+    // expect(
+    //   wrapper.contains('The purpose of the lesson is for people to learn'),
+    //   'purpose'
+    // ).to.be.true;
```

Comment:
in other cases, it may not make a lot of sense to test more deeply nested content in a shallow render test. for these, I would recommend following up with a test which mounts the LessonEditor component, since it is a top-level component, and then re-enabling these checks there.

Reply:
one trick with `mount` is that you probably want to mount the connected component, because you will need redux in order for it to be able to mount all of its children anyway. this becomes an integration test of sorts, even though it would probably still live in the unit test directory.